versions/3.1.1.md

```diff
@@ -3792,6 +3829,41 @@ security:
     - read:pets
 ```
 
+###### Security Requirement in a Referenced Document
+
+See [Resolving Implicit Connections](#resolvingImplicitConnections) for more information.
+
+Entry document `openapi.yaml`:
+
+```YAML
+components:
+  securitySchemes:
+    MySecurity:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+paths:
+  /foo:
+    $ref: "other.yaml#/components/pathItems/Foo"
+```
+
+Referenced document `other.yaml`:
+
+```YAML
+components:
+  securitySchemes:
+    MySecurity:
+      type: http
+      scheme: basic
+  pathItems:
+    Foo:
+      get:
+        security:
+        - MySecurity: []
```

Comment:
```suggestion
          MySecurity: []
```
Pattern field, not array item

Reply:
@ralfhandl array of objects with pattern field, not object with pattern field :-)
<img width="834" alt="Screen Shot 2024-06-11 at 8 01 14 AM" src="https://github.com/OAI/OpenAPI-Specification/assets/2358015/3e286b8b-f43f-45f3-9285-390b0e69b6a5">